public/app/features/trails/DataTrailsHistory.tsx

```diff
@@ -113,6 +123,22 @@ export class DataTrailHistory extends SceneObjectBase<DataTrailsHistoryState> {
         this.addTrailStep(trail, 'filters', parseFilterTooltip(urlState, filtersApplied));
         this.setState({ filtersApplied });
       }
+
+      // BUG: not currently picking up the dep env variable change
```

Comment:
Do we have an issue to track this bug?

Reply:
I made an issue in Scenes that shows in our board and we are tracking it here https://github.com/orgs/grafana/projects/516/views/1?pane=issue&itemId=79759856

Reply:
I see that issue was closed with a note from torkel. Do you mind applying the suggestion and deleting the comment here?

Reply:
Fixed! I applied it to the tests, Torkel's suggestion worked.

Reply:
Here is where I applied the fix https://github.com/grafana/grafana/pull/91221/commits/d1001272ebe4c59f52e6c722e1d1f578798ec4b2